cognee/infrastructure/databases/vector/milvus/MilvusAdapter.py

```
@@ -0,0 +1,247 @@
+    name = "Milvus"
+    url: str
+    api_key: Optional[str]
+    embedding_engine: EmbeddingEngine = None
+
+    def __init__(self, url: str, api_key: Optional[str], embedding_engine: EmbeddingEngine):
+        self.url = url
+        self.api_key = api_key
+
+        self.embedding_engine = embedding_engine
+
+    def get_milvus_client(self) -> "MilvusClient":
```

Ruff (0.8.0)

Undefined name `MilvusClient`

(F821)